DQM/BeamMonitor/plugins/Vx3DHLTAnalyzer.cc

```diff
@@ -340,7 +340,11 @@ int Vx3DHLTAnalyzer::MyFit(vector<double>* vals) {
       maxTransRadius = nSigmaXY * std::sqrt(std::fabs((*vals)[0]) + std::fabs((*vals)[1])) / 2.;
       maxLongLength = nSigmaZ * std::sqrt(std::fabs((*vals)[2]));
 
-      Gauss3D->Minimize();
+      try {
+        Gauss3D->Minimize();
+      } catch (...) {
```

Comment:
I think this construct is forbidden, or at least is flagged as such by the S/A

Reply:
Yes, rule 7.11 
https://cms-sw.github.io/cms_coding_rules.html

Reply:
Following the coding rules I can try something like:
```
if (!Gauss3D->Minimize()) {
   throw cms::Exception("FitFailed") << "Vx3DHLTAnalyzer \tInitial matrix not pos. def");
}
```
Would this be better?

Reply:
catching `cms::Exception` or std exceptions is allowed. What is not allowed if using `catch(...)`